README.md

```diff
@@ -18,7 +17,8 @@ run `npm i`
 
 #### `.env` requirements (where applicable)
 
-- `STORE`
+- `STORENAME`
+- `PORT`
 
 #### How to initialize/run your application (where applicable)
 
```

Comment:
On your image if you could remove the ; at the end then it will be imbedded on the page.